openapi/listings/listings.yaml

```
@@ -13,23 +13,23 @@ servers:
   - url: 'http://localhost:11001/products/listings'
     description: Localhost
 tags:
-  - name: Business Entity Listings
+  - name: Listing Sync Listings
   - name: Options
 paths:
-  /businessEntityListings:
+  /listingSyncListings:
     get:
       tags:
-        - Business Entity Listings
-      summary: List Business Entity Listings
-      operationId: list-businessListings
+        - Listing Sync Listings
+      summary: List Listing Sync Listings
+      operationId: list-listingSyncListings
       security:
         - OAuth2:
             - listing
       x-lifecycle:
-        status: proposed
+        status: trustedTester
       description: |-
-        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
-        Gets a list of listings for a business entity
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+        Gets a list of listings for a business location. This corresponds to the Listing Sync page in Listing Builder.
```

Can we clarify the type of listings?

```suggestion
        Gets a list of web directory listings for a business location. This corresponds to the Listing Sync page in Listing Builder.
```

---

We dont only deal in web directories, we also do GPS, aggregators, etc

---

Is there some collective term for them? "Digital directory"? If not that is fine.